JavaScript/JavaScriptSDK/Context/User.ts

```diff
@@ -29,12 +34,34 @@ module Microsoft.ApplicationInsights.Context {
          * The store region.
          */
         public storeRegion: string;
-        
+
+        static cookieSeparator: string = '|';
+
+        public setAuthId(id: string, accountId? : string) {
+            this.authId = id;
```

Comment:
This comment got me thinking about the types of input we may get from our users:
-	We can’t allow control characters – “;,= ” and our own “|” because it might mess with the cookie 
-	It’s unclear whether you can have non-ASCII characters in the cookie. This worked for me in chrome, FF and IE, but it’s not officially supported by the latest spec.

I decided to reject input with control characters, and encodeURI to avoid unicode characters.
(my initial thought was to encode in base64, But the out-of-the-box btoa() function isn’t supported in IE9 and earlier)